src/pages/Library.tsx

```diff
@@ -1,3 +1,8 @@
+/* QUESTION: How do u know what chapter it's on? Is there a bookmark?
```

Comment:
Answered on slack, so not sure if you still want an answer for this. We have ILibraryContext state stored in the library component.